CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
+- Added `whereConditions` config variable on Sage People component
```

Comment:
I'm wondering if `queryConditions` might be clearer, but I'm not concerned about it.